bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/VoiceManagerImpl.java

```diff
@@ -550,7 +550,7 @@ private void stopAllDialogs() {
     }
 
     private boolean checkLocales(@Nullable Set<Locale> supportedLocales, Locale locale) {
-        if (supportedLocales == null) {
+        if (supportedLocales == null || supportedLocales.isEmpty()) {
             // rule interpreter returns null so allowing it
```

Comment:
In case the comment is wrong (you said the rule interpreyer returns an empty list and not null), it had to be fixed or removed. 
I did not verified myself.

Reply:
Verified. We have no KS, STT or HLI services return null as supported locales.

Reply:
So please remove `@Nullable` from the parameter and remove the comment about rule interpreter.

Reply:
Thanks & done.